tensorboard/util/test_util.py

```diff
@@ -273,3 +275,40 @@ def ensure_tb_summary_proto(summary):
     return summary
 
   return summary_pb2.Summary.FromString(summary.SerializeToString())
+
+
+def _run_conditionally(guard, name, default_reason=None):
+  """Creates a decorator that skips a test when guard returns False.
```

Comment:
This function doesn’t actually create a decorator. It creates a function
that returns decorators. (This is a decorator factory factory.) The
distinction is important; “create a decorator” suggests that it would be
valid to write

```python
@run_v1_only  # not invoked!
def foo():
  pass
```

which is not the case.